src/did/did_doc.rs

```diff
@@ -0,0 +1,63 @@
+//! Set of interfaces that describe DID Document (https://www.w3.org/TR/did-core/)
+
+use serde_json::Value;
+
+/// Represents DID Document (https://www.w3.org/TR/did-core/)
+pub trait DIDDoc {
```

Comment:
I believe it would be great to have a `did` field here for completeness.

Reply:
I added it